SEC/Equities/EquityInstruments.rdf

```diff
@@ -406,12 +420,73 @@
+			</owl:Restriction>
+		</rdfs:subClassOf>
+		<rdfs:subClassOf>
+			<owl:Restriction>
+				<owl:onProperty rdf:resource="&fibo-sec-eq-eq;hasRedemptionPremium"/>
+				<owl:onClass rdf:resource="&fibo-fnd-acc-cur;MonetaryPrice"/>
+				<owl:minQualifiedCardinality rdf:datatype="&xsd;nonNegativeInteger">0</owl:minQualifiedCardinality>
+			</owl:Restriction>
+		</rdfs:subClassOf>
+		<rdfs:subClassOf>
+			<owl:Restriction>
+				<owl:onProperty rdf:resource="&fibo-sec-eq-eq;hasEarliestRedemptionDate"/>
+				<owl:onClass rdf:resource="&fibo-fnd-dt-fd;ExplicitDate"/>
+				<owl:minQualifiedCardinality rdf:datatype="&xsd;nonNegativeInteger">0</owl:minQualifiedCardinality>
+			</owl:Restriction>
+		</rdfs:subClassOf>
+		<rdfs:subClassOf>
+			<owl:Restriction>
+				<owl:onProperty rdf:resource="&fibo-sec-eq-eq;isRedeemableAtIssuerOption"/>
+				<owl:minQualifiedCardinality rdf:datatype="&xsd;nonNegativeInteger">0</owl:minQualifiedCardinality>
+				<owl:onDataRange rdf:resource="&xsd;boolean"/>
+			</owl:Restriction>
+		</rdfs:subClassOf>
+		<rdfs:subClassOf>
+			<owl:Restriction>
+				<owl:onProperty rdf:resource="&fibo-sec-eq-eq;isRedeemableAtShareholderOption"/>
+				<owl:minQualifiedCardinality rdf:datatype="&xsd;nonNegativeInteger">0</owl:minQualifiedCardinality>
+				<owl:onDataRange rdf:resource="&xsd;boolean"/>
+			</owl:Restriction>
+		</rdfs:subClassOf>
+		<rdfs:label xml:lang="en">equity redemption provision</rdfs:label>
+		<skos:definition xml:lang="en">redemption provision that specifies the conditions under which the issuer or shareholder may redeem the shares</skos:definition>
+	</owl:Class>
+	
+	<owl:Class rdf:about="&fibo-sec-eq-eq;ExtendablePreferredShare">
+		<rdfs:subClassOf rdf:resource="&fibo-sec-eq-eq;RedeemablePreferredShare"/>
+		<rdfs:label>extendable preferred share</rdfs:label>
+		<skos:definition>redeemable preferred share whose redemption date can be extended</skos:definition>
+		<skos:editorialNote>Need to add hasExtensionProvision, once that is moved from Debt</skos:editorialNote>
+	</owl:Class>
+	
 	<owl:Class rdf:about="&fibo-sec-eq-eq;ExtraordinaryVotingRight">
 		<rdfs:subClassOf rdf:resource="&fibo-sec-eq-eq;VotingRight"/>
 		<rdfs:label>extraordinary voting right</rdfs:label>
 		<skos:definition>right to vote at extraordinary meetings, as opposed to being able to vote strictly at annual general meetings</skos:definition>
 	</owl:Class>
 	
+	<owl:Class rdf:about="&fibo-sec-eq-eq;FixedRateDividend">
+		<rdfs:subClassOf rdf:resource="&fibo-sec-eq-eq;PreferredDividend"/>
+		<rdfs:subClassOf>
+			<owl:Restriction>
+				<owl:onProperty rdf:resource="&fibo-sec-eq-eq;hasFixedDividendRate"/>
+				<owl:someValuesFrom rdf:resource="&fibo-fnd-acc-cur;PercentageMonetaryAmount"/>
+			</owl:Restriction>
+		</rdfs:subClassOf>
+		<rdfs:label>fixed rate dividend</rdfs:label>
+		<skos:definition>dividend that provides a specified annual return on the nominal value (and any premium) paid on shares</skos:definition>
```

Comment:
I don't understand the "(and any premium)". If the fixed rate is 10% and I paid $100 with $10 premium does that mean I get $11?
Separately is the interest compounded or it based strictly on what was originally paid?
Finally there is a huge difference between "nominal value" and "amount paid" on shares! The note seems to acknowledge that but the definition needs fixing.

Reply:
The premium, if there is one, is specified as a provision of the contract.  Interest doesn't apply, and for preferred shares, the dividend is calculated based on the nominal value at issuance, not the cumulative face value of the share.